projects/plugins/crm/includes/ZeroBSCRM.MetaBoxes3.Transactions.php

```diff
@@ -216,6 +216,41 @@ public function html( $transaction, $metabox ) {
 						<td colspan="2"><hr /></td>
 					</tr>
 
+					<?php
+
+					// Transaction Status
+					$potential_statuses = zeroBSCRM_getTransactionsStatuses( true );
+					$current_status     = '';
+				if ( is_array( $transaction ) && isset( $transaction['status'] ) ) {
+						$current_status = $transaction['status'];
+				}
+				?>
+					<tr class="wh-large">
+						<th><label for="zbst_status"><?php echo esc_html( __( 'Transaction Status:', 'zero-bs-crm' ) ); ?></label>
+						</th>
+						<td>
+						<select id="zbst_status" name="zbst_status">
+							<?php
+						foreach ( $potential_statuses as $status ) {
+							$sel = $status === $current_status ? ' selected' : '';
+							echo '<option value="' .
+							esc_attr( $status ) .
+							'"' .
+							esc_attr( $sel ) .
+							'>' .
+							esc_html(
+								sprintf(
+									/* Translators: placeholder is the name of the transaction status */
+									__( '%s', 'zero-bs-crm' ), // phpcs:ignore WordPress.WP.I18n.NoEmptyStrings
```

Comment:
I feel like there should be a better way to escape / internationalize this - with the way I've written it (including only the placeholder) the linter was complaining with the WordPress.WP.I18n.NoEmptyStrings warning.

Reply:
Ideally, yes. But our transaction status strings are currently broken. Context: https://github.com/Automattic/zero-bs-crm/issues/1497

Reply:
As transactions are user-configurable, translation shouldn't be added here. A translator can't translate a standalone `%`. :^)

I believe upon setup the default transaction statuses are translated, though I don't recall if that's truly the case or not.

Reply:
Ok, I've changed this to just 'esc_html( $status )' now.